### PR TITLE
MAD-306 - cancel account

### PR DESCRIPTION
**Pull request by gothlike**

### CODE REVIEW

File: inbox/api/ns_api.py [hunk 1]
```diff
@@ -249,6 +251,27 @@ def one_account():
...
+@app.route('/account', methods=['DELETE'])
+def account_delete():
+    account = g.namespace.account
+
+    account.disable_sync("account deleted")
+
+    g.db_session.commit()
+
+    try:
+        gevent.sleep(3)
```
Comment (gothlike) [inline inbox/api/ns_api.py:263]: An ugly spike: waiting for marking account for deletion

File: inbox/models/util.py [hunk 2]
```diff
@@ -131,7 +131,7 @@ def batch_delete_namespaces(ids_to_delete, throttle=False,
              time=end - start, count=deleted_count)
 
 
-def delete_namespace(namespace_id, throttle=False, dry_run=False):
+def delete_namespace(account_id, namespace_id, throttle=False, dry_run=False):
```
Comment (gothlike) [inline inbox/models/util.py:134]: Fixed function signature. Otherwise deletion doesn't work properly

File: inbox/api/ns_api.py (hunk 1 above)
Comment (iderets) [reply inbox/api/ns_api.py:263]: are you sure that 3 sec will be ok here? may be there is another way to check it?

**Review by iderets — APPROVED** [review]

File: inbox/api/ns_api.py (hunk 1 above)
Comment (gothlike) [reply inbox/api/ns_api.py:263]: It's enough for now. Later on it must be moved to separate jobs:
- first one is executed on demand (user requests deletion of account)
- second one is executed with some schedule, say, once in an hour (actual deletion of account)

I'll create put a task in backlog.

But for now - I'm sure it's enough

**Pull request merged**